app/scripts/modules/netflix/serverGroup/awsServerGroupDetails.html

```diff
@@ -282,15 +282,25 @@ <h3 select-on-dbl-click>
     <collapsible-section heading="Edda Links">
       <ul>
         <li>
-          <a target="_blank" ng-href="http://edda-main.{{ctrl.serverGroup.region}}.{{ctrl.serverGroup.account}}.netflix.net:7001/REST/v2/aws/autoScalingGroups/{{ctrl.serverGroup.name}}">
+          <a target="_blank" ng-href="http://edda-{{ctrl.serverGroup.accountDetails.accountId}}.{{ctrl.serverGroup.region}}.{{ctrl.serverGroup.accountDetails.environment}}.netflix.net:7001/REST/v2/aws/autoScalingGroups/{{ctrl.serverGroup.name}}">
```

Comment:
This actually makes these links work in the different accounts.

Previously they were busted in mce (_for example_).